src/node/rpc/member_frontend.h

```diff
@@ -1594,13 +1594,13 @@ namespace ccf
           auto s = sig_view->get(0);
           if (s)
           {
-            ma->state_digest =
-              std::vector<uint8_t>(s->root.h.begin(), s->root.h.end());
-
+            ma->state_digest = s->root.hex_str();
             ma_view->put(caller_id, ma.value());
           }
+          nlohmann::json j;
+          j["state_digest"] = ma->state_digest;
 
-          return make_success(ma->state_digest);
+          return make_success(j);
```

Comment:
Is it possible to do `return make_success(ma->state_digest);` instead of constructing a JSON object?

Reply:
No, because the serialisation auto-unwraps state_digest, so that ends up being just a string.